invenio_files_rest/__init__.py

```diff
@@ -386,6 +387,14 @@
 Be aware that there are security implications to take into account when
 serving files. See the :ref:`usage-security` for more information.
 
+Invenio-Files-Rest provides also the functionality to serve your files directly
+from your external storage. This is achieved by attaching the
+`X-Accel-Redirect` header to the response, which will then be redirected by
```

Comment:
Comment: is there any good resource we can point to that documents this header? If that exists I would appreciate to have a link and learn it.